documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc

```diff
@@ -12,11 +12,14 @@ You can use a standalone Topic Operator with a Kafka cluster that is not managed
 A standalone deployment can operate with any Kafka cluster.
 
 Standalone deployment files are provided.
-Configure the `05-Deployment-strimzi-topic-operator.yaml` deployment file to add the environment variables that enable the Topic Operator to connect to a Kafka cluster.
+Edit the `05-Deployment-strimzi-topic-operator.yaml` deployment file to add the environment variables that enable the Topic Operator to connect to a Kafka cluster.
```

Comment:
I think this sounds a bit weird in this place. It sounds like a command, but it does not have the details to execute on it. Also, the user does not necessarily need to add any environment variables. Just set the existing ones to the right values.

Reply:
Changed for both operators:

> Standalone deployment files are provided with Strimzi.
> Use the `05-Deployment-strimzi-topic-operator.yaml` deployment file to deploy the Topic Operator.
> Add or set the environment variables needed to make a connection to a Kafka cluster.